app/admin/routing/routing_plans.rb

```diff
@@ -7,14 +7,11 @@
   acts_as_clone
   acts_as_safe_destroy
   acts_as_async_destroy('Routing::RoutingPlan')
-  acts_as_async_update('Routing::RoutingPlan',
-                       lambda do
-                         {
-                           sorting_id: Sorting.pluck(:name, :id),
-                           use_lnp: boolean_select,
-                           rate_delta_max: 'text'
-                         }
-                       end)
+  begin
+    acts_as_async_update BatchUpdateForm::RoutingPlan
```

Comment:
```suggestion
    acts_as_async_update 'BatchUpdateForm::RoutingPlan'
```
class should be provided as string to prevent loading class while booting app